src/Nethermind/Nethermind.Specs/ChainSpecStyle/Json/ChainSpecJson.cs

```diff
@@ -157,6 +156,10 @@ internal class AuraEngineJson
 
             public IDictionary<long, IDictionary<Address, byte[]>> RewriteBytecode => Params.RewriteBytecode;
 
+            public Address WithdrawalContractAddress => Params.WithdrawalContractAddress;
+
+            public ForkActivation WithdrawalContractTransition => Params.WithdrawalContractTransition;
```

Comment:
@LukaszRozmej @jmederosalvarado Do we need `ForkActivation[] WithdrawalContractTransitions` as well or a single one is enough? I need more info on that in general.

Reply:
no need for withdrawal contract transition as an aura param. see other comment regarding this.

Reply:
We don't need it unless we want to upgrade withdrawal contract to different address - YAGNI for now.